e2e/tests/events.bats

```diff
@@ -10,36 +10,66 @@ load variables
 }
 
 @test "Create node function with schema" {
-    run dispatch create function nodejs6 node-hello-subscribe-${RUN_ID} ${DISPATCH_ROOT}/examples/nodejs6/hello.js
+    run dispatch create function nodejs6 node-echo-back ${DISPATCH_ROOT}/examples/nodejs6/debug.js
     echo_to_log
     assert_success
 
-    run_with_retry "dispatch get function node-hello-subscribe-${RUN_ID} --json | jq -r .status" "READY" 6 5
-    sleep 15 # https://github.com/vmware/dispatch/issues/67
+    run_with_retry "dispatch get function node-echo-back --json | jq -r .status" "READY" 8 5
 }
 
 @test "Create subscription" {
-    run dispatch create subscription test.event node-hello-subscribe-${RUN_ID}
+    run dispatch create subscription --name testsubscription --event-type test.event node-echo-back
     echo_to_log
     assert_success
 
-    run_with_retry "dispatch get subscription test_event_node-hello-subscribe-${RUN_ID} --json | jq -r .status" "READY" 4 5
+    run_with_retry "dispatch get subscription testsubscription --json | jq -r .status" "READY" 4 5
 }
 
 @test "Emit event" {
-    run dispatch emit test.event --payload='{"name": "Jon", "place": "Winterfell"}'
+    sleep 5 # Still seeing the issue https://github.com/vmware/dispatch/issues/67
```

Comment:
You've been removing this everywhere. Why add it here?

Reply:
This is the only place where, after removing the sleeps, I've noticed the issue again.